composer.json

```diff
@@ -40,7 +40,8 @@
         "sumocoders/framework-search-bundle": "^3.0",
         "sumocoders/framework-example-bundle": "^4.0",
         "sumocoders/framework-multi-user-bundle": "^4.0",
-        "knplabs/knp-paginator-bundle": "^2.5"
+        "knplabs/knp-paginator-bundle": "^2.5",
+        "simple-bus/symfony-bridge": "^4.1"
```

Comment:
@carakas according to the documentation, the added bundle already includes the doctrine-orm-bundle (https://github.com/SimpleBus/SymfonyBridge/blob/master/README.md). So no need to include it as well.